lib/manager/terraform/extract.spec.ts

```diff
@@ -16,8 +21,8 @@ describe('lib/manager/terraform/extract', () => {
     it('extracts', () => {
       const res = extractPackageFile(tf1);
       expect(res).toMatchSnapshot();
-      expect(res.deps).toHaveLength(44);
-      expect(res.deps.filter((dep) => dep.skipReason)).toHaveLength(8);
+      expect(res.deps).toHaveLength(39);
```

Comment:
Why is this changed? Adding functionality shouldn't normally decrease the number of dependencies detected

Reply:
Indeed. After some more debugging I realized using the same terraform manager to cover terragrunt is not going to work, so I'm working on setting up a new manager.